lib/node_modules/@stdlib/math/base/complex/divide/benchmark/cdiv_smith.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+// MODULES //
+
+var abs = require( '@stdlib/math/base/special/abs' );
+
+// MAIN //
+
+/**
+* Divides two complex numbers.
```

Comment:
Should I benchmark this version and reference the same paper (which describes it)?